src/ApplicationManager.cpp

```
@@ -1206,6 +1269,36 @@ void ApplicationManager::handleKeypadCancel() {
+
+void ApplicationManager::updateTrayDashboard(const TrayDashboardState& state) {
+    trayDashboardState_ = state;
+}
+
+const TrayDashboardState& ApplicationManager::getTrayDashboardState() const {
+    return trayDashboardState_;
+}
+
+void ApplicationManager::handleTrayUpdate() {
+    // Persist to NVS
+    Preferences prefs;
```

Clang (14.0.6)

[warning] variable 'prefs' is not initialized

(cppcoreguidelines-init-variables)